app/assets/stylesheets/_buttons.scss

```diff
@@ -34,6 +34,11 @@
   }
 }
 
+.add-to-vocab-sheet {
```

Comment:
Just did some test on this locally and this whole style isnt actually needed. The card__button class overrides the float: left with its own, and the font size shouldnt be made larger based on what staging and prod look like.